rules/autoconsent/monzo-com.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "monzo.com",
+    "prehideSelectors": [".cookie-alert"],
```

Comment:
I think we need to make it more specific too, so we don't occasionally hide unhandled popups

Reply:
see comment below: https://github.com/duckduckgo/autoconsent/pull/22#issuecomment-1204897857